rts/Rml/SolLua/RmlSolLua.cpp

```diff
@@ -0,0 +1,55 @@
+﻿// This file is part of the Spring engine (GPL v2 or later), see LICENSE.html
```

Comment:
This should imho contain additional information that it's derived from MIT licensed file copyrighted by xyz.